packages/terra-button/CHANGELOG.md

```diff
@@ -3,6 +3,17 @@ Changelog
 
 Unreleased
 ----------
+* See Release Notes for Migration Guide
+
+### Added
+* refCallback, isIconOnly, onBlur, onFocus, onKeyDown, onKeyUp props
+
+### Changed
+* text prop is now required for accessibility purposes
+* variant options changes from `primary`, `secondary`, or `link` to `neutral`,  `emphasis`, `de-emphasis`, `utility` or `action`
+
+### Removed
+* size and isCompact props
 
```

Comment:
This is just preference, the changed section takes on the role of added and removed so it becomes a little ambiguous. It might be worth considering splitting them out into what was added and what was removed. 

### Added
* refCallback, isIconOnly, onBlur, onFocus, onKeyDown, onKeyUp props
* `neutral`,  `emphasis`, `de-emphasis`, `utility` and `action` variants

### Removed
* size prop
* isCompact prop
* `primary`, `secondary`,  and `link` variants

### Changed
* text prop is now required to meet accessibility